devon4ng-architecture/files/core-shared-modules.md

```diff
@@ -0,0 +1,7 @@
+You can switch to the IDE tab to checkout the files of the application you just cloned.
+
+* If you check out the file 'src > app > core > core.module.ts', you can see services like `TranslocoRootModule` (translation service), `HttpRequestInterceptorService` (HTTP interceptor sesrvice) and Angular Material related modules being part of this core module. This core module is then imported into every other module as the services are used throughout the application.
```

Comment:
You can use https://github.com/devonfw-tutorials/tutorial-compiler/blob/main/documentation/Functions.md#openfile here